server.js

```diff
@@ -1,16 +1,63 @@
+import cors from 'cors';
 import express from 'express'
+import passport from 'passport';
+import session from 'express-session';
+import faqRouter from './routes/faq.js';
 import userRouter from './routes/user.js';
 import authRouter from './routes/auth.js';
-import faqRouter from './routes/faq.js';
+import incomeRouter from './routes/income.js';
+import expenseRouter from './routes/expense.js';
+import accountsRouter from './routes/account.js';
+import { config } from 'dotenv';
+import { jwtCallback } from './utils/passport.js';
+import { ExtractJwt, Strategy } from 'passport-jwt';
+
+config();
+
+const opts = {
+    jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
+    secretOrKey: process.env.JWT_SECRET,
+}
+
+const app = express();
+
+passport.use(new Strategy(opts, jwtCallback));
+
+passport.serializeUser(function (user, done) {
```

Comment:
everywhere you are using arrow function for callbacks. so why there is no ?

Reply:
fixed